nodes/stampzilla-server/nodeserver.go

```diff
@@ -75,28 +84,86 @@ func (ns *NodeServer) newNodeConnection(connection net.Conn) {
 			name = node.Name
 			uuid = node.Uuid
 
-			if logicChannel == nil {
-				logicChannel = ns.Logic.ListenForChanges(uuid)
```

Comment:
I would like to still have one logic listener per node connection. It will handle multiple nodes sending updates at the same time better. So using ListenForChanges instead of your new Listen. 
That way the UpdatePackage does not need to contain a Uuid and can be string instead of a struct. (like it was before).  The logic.Update is still nice so we have the marshaling there.